src/test/scala/com/cognite/sdk/scala/v1/FunctionsTest.scala

```diff
@@ -0,0 +1,85 @@
+package com.cognite.sdk.scala.v1
+
+import io.circe.Json
+import org.scalatest.Inspectors._
+import io.circe.JsonObject
+import com.cognite.sdk.scala.common._
+
+class FunctionsTest extends SdkTestSpec with ReadBehaviours {
+  override lazy val client: GenericClient[cats.Id] = GenericClient.forAuth[cats.Id](
+    "playground", auth, apiVersion = Some("playground"))(implicitly, sttpBackend)
+
```

Comment:
Use the standard resource tests, like `it should behave like readable(client.functions)`

Reply:
I'm not sure that would work since functions don't support limit and cursor. https://docs.cognite.com/api/playground/#tag/Functions